pdr_backend/lake/duckdb_data_store.py

```diff
@@ -349,3 +351,101 @@ def get_query_move_table_data(self, from_table, to_table):
         return (
             f"INSERT INTO {to_table.table_name} SELECT * FROM {from_table.table_name};"
         )
+
+    @enforce_types
+    def export_tables_to_parquet_files(
+        self,
+        seconds_between_exports: int,
+        number_of_files_after_which_combine_into_one: int,
```

Comment:
i don't understand how this is used... parquet is designed to have hundreds of files existing in the same folder, and you should be able to just query the folder

please revise this answer

Reply:
will remove the combine part but will keep the param under another name. We can use it to nuke everything and re-export the db as 1 file periodically. Once a lot of files are created it could impact the performance.